docs/framework/server-side-editor-api.md

```diff
@@ -0,0 +1,336 @@
+---
+category: framework
+order: 500
+meta-title: Server-Side Editor API | CKEditor 5 Documentation
+modified_at: 2025-05-20
+---
+
+# Server-side editor API
+
+Server-Side Editor API allows for deep and complex integration of your application with document data, enabling you to manipulate content and manage collaborative data such as suggestions, comments, and revision history directly from your server-side code.
+
+## Why use server-side editor API?
+
+While CKEditor 5 provides a rich client-side editing experience, there are many scenarios where server-side content processing is essential:
+
+* **Security**: Process sensitive content in a controlled environment without exposing it to client-side manipulation.
+* **Performance**: Handle large-scale content operations without impacting the user's browser.
+* **Consistency**: Ensure uniform content changes across multiple documents.
+* **Integration**: Connect with other server-side systems and databases directly.
+* **Automation**: Run content processing tasks as part of your server workflows.
+* **Scalability**: Process multiple documents simultaneously without client-side limitations.
+
+## Common use cases
```

Comment:
This section feels a bit too long to me. We already covered it partially in previous sections (although in different words). And then the next sections are more useful and still present various common use cases.

I wonder if this section is needed at all, but it can be a good introduction, just don't keep the reader for too long from the "meat". Maybe choose 5 points from these proposed here?

Reply:
Okay, I removed:
* Bulk content updates - demonstrated later in the "Using commands" section.
* Dynamic content - overlaps with automation.
* Automated revision control - covered in "Working with revision history" section.
* Custom integration - too generic.
* Automatic checkpoints - too specific.

Let me know if you want to change something here.